docs/upgrade/upgrade-security.asciidoc

```diff
@@ -11,13 +11,13 @@ NOTE: For detailed information about the 8.0 release, check out the <<release-no
 
 [discrete]
 [[upgrade-8.1.1]]
-== Upgrade to 8.1.1
+== Avoid upgrading to 8.1.1
```

Comment:
Revised since the heading seemed to say "do this" when actually the admonition below says "don't do this."